### PR TITLE
Merge from regression2

### PR DESCRIPTION
**Pull request by diegolovison**

### CODE REVIEW

**Comment by diegolovison** [comment]
Please, do not merge this PR yet

File: core/pom.xml [hunk 1]
```diff
@@ -14,7 +14,7 @@
 
    <properties>
       <version.log4j>1.2.16</version.log4j>
-      <version.log4j2>2.3</version.log4j2>
+      <version.log4j2>2.8.2</version.log4j2>
```
Comment (alanfx) [inline core/pom.xml]: 2.11.0 seems to the the latest version - https://logging.apache.org/log4j/2.x/download.html

File: plugins/infinispan91/pom.xml [hunk 2]
```diff
@@ -16,6 +15,7 @@
 
    <properties>
       <version.infinispan>${infinispan91.infinispan.version}</version.infinispan>
+      <version.narayanajta>5.0.4.Final</version.narayanajta>
```
Comment (alanfx) [inline plugins/infinispan91/pom.xml]: I removed this in https://github.com/radargun/radargun/pull/518, and you can remove this file from the PR

File: plugins/infinispan92/pom.xml [hunk 3]
```diff
@@ -15,6 +15,7 @@
 
    <properties>
       <version.infinispan>${infinispan92.infinispan.version}</version.infinispan>
+       <version.narayanajta>5.0.4.Final</version.narayanajta>
```
Comment (alanfx) [inline plugins/infinispan92/pom.xml]: I removed this in https://github.com/radargun/radargun/pull/518, and you can remove this file from the PR

File: plugins/jdg72/pom.xml [hunk 4]
```diff
@@ -15,6 +15,9 @@
 
    <properties>
       <version.infinispan>${jdg72.infinispan.version}</version.infinispan>
+      <version.infinispan.public>8.2.1.Final</version.infinispan.public>
+      <version.narayanajta>5.2.17.Final-redhat-1</version.narayanajta>
+      <version.jta-spec>1.0.1.Final-redhat-2</version.jta-spec>
```
Comment (alanfx) [inline plugins/jdg72/pom.xml]: I removed this in https://github.com/radargun/radargun/pull/518, and you can remove this file from the PR

**Comment by diegolovison** [comment]
Done

File: plugins/jdg72/pom.xml [hunk 5]
```diff
@@ -29,6 +29,16 @@
             <version.infinispan>8.5.0.ER3-redhat-1</version.infinispan>
          </properties>
       </profile>
+      <profile>
+         <id>demos</id>
```
Comment (alanfx) [inline plugins/jdg72/pom.xml]: Sorry, this profile should go away as well. This was related to the dist-exec demo JAR file that was removed

File: core/pom.xml (hunk 1 above)
Comment (diegolovison) [reply core/pom.xml]: https://github.com/radargun/radargun/issues/525

**Comment by alanfx** [comment]
@diegolovison now you've gone too far with removing the `demos` profile. I only meant for you to remove it in the `jdg72`, `infinispan92`, and `-snapshot` plugins. The JAR does exist for the rest of the plugins.

**Pull request merged**